package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "start":"xdg-open http://localhost:3000 && node server.js"
```

Comment:
Reminder that this isn't cross-platform; you'll probably want to `npm install --save-dev opn-cli` instead so you can use `opn` here.